net/grpc/gateway/examples/grpc-web-java/greeter-service/src/main/proto/greeter.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+option java_package = "grpcweb.examples.greeter";
+
+package grpcweb.examples.greeter;
+
+// The greeting service definition.
+service Greeter {
```

Comment:
There is a copy here: https://github.com/grpc/grpc-web/blob/master/net/grpc/gateway/examples/helloworld/helloworld.proto (without the `package`). Is there someway we can reuse that copy of the `helloworld.proto`?

Reply:
Yes it is duplicate.
add a comment to to it to say it is a copy.

It should be possible to reuse the same proto but maven is not allowing me to refer to files in parent directories. Have to figure that out.

Reply:
I see. Yea we can improve this through iterations. Ideally we should avoid duplicating this.

Reply:
adding a TODO.